.github/workflows/acceptance-test.yml

```diff
@@ -29,6 +29,26 @@ jobs:
     steps:
       - name: Base checkout
         uses: actions/checkout@v4
+        if: |
+          github.event_name != 'repository_dispatch' &&
+          (
+            github.ref == 'refs/heads/main' || 
+            (
+              github.event_name == 'pull_request' && 
+              github.event.pull_request.head.repo.full_name == github.repository
+            )
+          )
```

Comment:
What is the reason behind avoiding this checkout when both` github.event_name == 'repository_dispatch' ` and `github.ref == 'refs/heads/main'` happen?

Reply:
Do we ever have the intention of running manual workflows on our branches? 🤔 
From my understanding we only do this on forked PRs (where we trigger manual workflows), therefore we shouldn't use this if we trigger workflows in other scenarios.

Reply:
Yeah, maybe not, the only case I had in mind is when running a GitHub action directly from the Actions panel. We don't have that yet but maybe we'll include it. But even that I think it just uses the workflow_dispacth event and not the repo_dispatch one.